specs/card_spec.rb

```diff
@@ -0,0 +1,46 @@
+
+require_relative 'spec_helper'
+
+describe Card do
+  describe "You can create a Card instance" do
+
+    it "Can be created" do
+      [:hearts, :spades, :clubs, :diamonds].each do |suit|
+        (1..13).each do |value|
+          card = Card.new(value, suit)
+          card.must_be_instance_of Card
+        end
+      end
+    end
+
+    it "Raises an ArgumentError for invalid parameters" do
+      proc {Card.new 0, :diamonds}.must_raise ArgumentError
+    end
+  end
+
+  describe "Testing to_s" do
+
+    it "to_s returns a readable String value logically for values 2-9" do
+      # Test to ensure that to_s works for cards values 2-10
+      # for example:  "2 of diamonds"
+    end
+
+    it "to_s returns a readable String value for Ace, Jack, Queen, King" do
+      # Test to ensure that to_s works for cards values 1, and 11-13
+      # For example: "Queen of hearts"
+    end
+  end
+
+  describe "Reader methods" do
```

Comment:
I'm not sure if there should.... but should there be tests that ensure you can't set the values of these read-only attributes?